pkg/plugins/platform/local.go

```diff
@@ -498,3 +498,8 @@ func (l *LocalApp) AddHostsEntry() error {
 	err = system.RunCommandPipe("sudo", hostnameArgs)
 	return err
 }
+
+// constructContainerName builds a container name given the type (web/db/dba) and the appName
+func constructContainerName(containerType string, appName string) string {
```

Comment:
I'd really like to see us use `FindContainerByLabels()` here and then `util.ContainerName()` on the result.

This hasn't been the case to date, but I would really like to see us be deliberate and consistent about the way we interact with containers.